source/about/mattermost-v10-changelog.md

```diff
@@ -3,6 +3,94 @@
+
+### Bug Fixes
+ - Fixed various issues on the "Create Team" screen.
+ - Fixed several accessibility issues across the login process, account creation, and MFA setup.
+ - Fixed an issue where horizontal rule (HR) elements were not visible in preview mode in the right-hand sidebar (RHS).
+ - Fixed an issue with inconsistent sizing of markdown images in preview mode.
+ - Fixed a keyboard navigation issue within thread items.
+ - Fixed layout issues with the emoji picker on mobile browsers.
+ - Fixed an issue with the positioning of “edited” text and tooltips in certain scenarios.
+ - Fixed the accessibility of the search box.
+ - Fixed an issue where post list scrolling didn’t work when pressing the "Page Up" or "Page Down" keys.
+ - Fixed issues with screen reader support in the **Threads** view.
+ - Fixed keyboard navigation issues in the **Threads** view.
+ - Fixed accessibility issues in the **Invite** modal.
+ - Fixed various accessibility issues in the **Browse Channels** modal.
+ - Fixed an issue that prevented team admin permissions from being modified when missing in the "All Members" section.
+ - Fixed possible deadlocks when updating ``SidebarCategories`` and ``SidebarChannels`` tables.
+
+### config.json
+New setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+ - Under ``SupportSettings`` in ``config.json``:
+    - Added ``ReportAProblemType``, ``ReportAProblemMail``, ``AllowDownloadLogs`` configuration settings to enhance the behavior for reporting issues in the platform.
+
+#### Changes to Enterprise plans:
+ - Under ``ExperimentalAuditSettings`` in ``config.json``:
+    - Added ``Certificate`` configuration setting to accept a certificate to be used for audit logging egress.
+ - Under ``LdapSettings`` in ``config.json``:
+    - Added ``ReAddRemovedMembers`` configuration setting to add a LDAP setting to re-add removed members.
+
+### Open Source Components
+ - Added ``monaco-editor`` and ``monaco-editor-webpack-plugin``, and removed ``dynamic-virtualized-list``, ``popper.js``, ``react-hot-loader``, ``react-popper`` from https://github.com/mattermost/mattermost.
+
+### Go Version
+ - v10.9 is built with Go ``v1.23.7``.
+
+### Known Issues
+ - Setting the license file location through an environment variable still gives the option to upload a new license through the System Console, resulting in the license being overwritten by the one set through the environment variable. See this [knowledge base article](https://support.mattermost.com/hc/en-us/articles/33911983851284-System-console-still-displays-old-license-after-uploading-a-new-one) on how to resolve this issue.
+ - Searching stop words in quotation marks with Elasticsearch enabled returns more than just the searched terms.
+ - Slack import through the CLI fails if email notifications are enabled.
```

Comment:
Can you remind me of the criteria for including known issues in this list? I feel like our entire open backlog could qualify here, and the ones listed here don't feel particularly high-impact.

Should we just remove this section?

Reply:
It's normally for new regressions in a release that were not fixed in time. I've removed everything else expect the first known issue as it was previously requested to be added by the CS team.